scripts/excelToJson.py

```diff
@@ -4,17 +4,24 @@
 from pathlib import Path
 import logging
 from typing import Union, List, Any, Dict
-
 import numpy as np
 import pandas as pd
 from flatten_json import flatten
 
-EXPECTED_COLUMNS = ["RS Number", "Required Standard", "Ref Calculation", "Additional info", "Basic", "Normal"]
+"""
+IVA-Defects excel spreadsheet needs to exist in working directory. Change global excel_file to match actual file name.
```

Comment:
Why is this doc string here?

Reply:
Going to change it to a proper commen syntax, but just to inform anyone else who needs to use it

Reply:
Maybe it would be more appropriate to put it in the read me file or something